src/metabase/models/user.clj

```diff
@@ -213,6 +213,13 @@
        (dissoc :password)
        (assoc :ldap_auth true))))
 
+(s/defn update-user!
+  "Update the first and last name of a user created with SSO, in case the name returned by the SSO provider has changed"
+  [user-id first-name last-name]
+  (db/update! User user-id
+              :first_name first-name
+              :last_name last-name))
```

Comment:
This function doesn't do anything special on top of what `db/update! User` already does... I'd take it out because it's just making things confusing when you come across it. Someone who comes across it might wonder why some code uses `db/update! User` directly, and some code uses `update-user!`; and which of the two should they use?

One of the goals of Toucan is to eliminate the need for functions like `update-user!` in the first place, since `db/update!` is just a more generate update function, and you can customize its behavior for a given model with `pre-update` or `post-update`